test/e2e/profiles/oidc/.kratos.yml

```diff
@@ -45,10 +42,12 @@ selfservice:
     registration:
       ui_url: http://localhost:4455/registration
       after:
+        password:
+          hooks:
+            - hook: session
```

Comment:
In the `spa` profile we always have a session after registration. Just added it here to the `oidc` profile used by the express app.